packages/terra-avatar/src/terra-dev-site/doc/avatar/Facility.3.doc.mdx

```diff
@@ -0,0 +1,46 @@
+import { Badge } from '../../../../package.json?dev-site-package';
+
+import Facility from '../example/facility/Facility?dev-site-example';
+import FacilityImage from '../example/facility/FacilityImage?dev-site-example';
+import FacilitySize from '../example/facility/FacilitySize?dev-site-example';
+import FacilityColorVariants from '../example/facility/FacilityColorVariants?dev-site-example';
+
+import FacilityPropsTable from '../../../../src/variants/Facility?dev-site-props-table';
+
+# Facility
```

Comment:
It looks like we copied over the content from `docs/f.md/.`Do we need to delete `'../../../../docs/facility.md';`?

Reply:
updated here https://github.com/cerner/terra-core/pull/2810/commits/14dc3d156f17f1c1a07ad277088e105d5a9c9fe6